Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/stateselection/FederalStateSelectionFragment.kt

```diff
@@ -60,15 +63,15 @@ class FederalStateSelectionFragment : Fragment(R.layout.federal_state_selection_
                     is FederalStateSelectionViewModel.Events.OpenDistricts -> findNavController().navigate(
                         FederalStateSelectionFragmentDirections
                             .actionFederalStateSelectionFragmentToFederalStateSelectionFragment(
-                                it.selectedFederalStateShortName
+                                it.selectedFederalState.number
                             )
                     )
                 }
             }
         }
     }
 
-    private fun getToolbarLabel() = if (navArgs.selectedFederalStateShortName != null) {
+    private fun getToolbarLabel() = if (navArgs.selectedFederalState != -1) {
```

Comment:
Why `-1` and not nullable parameter as before?

Reply:
i wanted to avoid matching the enum via the string value and ints cant be nulled afaik

Reply:
`int` can't be null, `string` can be, we can also use nullable `Parcelable`  object but it's not a big deal, maybe just define `-1` as some constant to be clear what are you comparing there